ci/run_vagrant_install_test.yml

```diff
@@ -104,6 +120,38 @@
       debug:
         msg: "{{ setup_harvester_result.stdout_lines }}"
 
+    - name: Clear the previous temp file
+      shell: >
+        ./cleanup_test_files.sh
+      args:
+        chdir: "{{ WORKSPACE }}/harvester-installer/ci/terraform"
+
+    - name: Testing basic operations with terraform
+      register: terraform_test
+      shell: |
+        curl https://releases.hashicorp.com/terraform/1.3.7/terraform_1.3.7_linux_amd64.zip -o terraform_bin.zip
+        unzip -o terraform_bin.zip
+        ./get_kubeconfig.sh 192.168.0.30 192.168.0.131
```

Comment:
Can we get it via the env variable? also need to doc where the node exists.

Reply:
Sure. This IP/VIP is from the `settings.yml` on the `ipxe-example` repo.
I change the way to load it from the above file. WDUT?